src/plugins/dashboard/public/application/utils/use/use_saved_dashboard_instance.ts

```diff
@@ -44,81 +44,72 @@ export const useSavedDashboardInstance = (
 
     const getSavedDashboardInstance = async () => {
       try {
-        let savedDashboardInstance: any;
+        let dashboardInstance: any;
```

Comment:
One todo is that we should define the type here for dashboardInstance, i think we already have a DashboardSavedObject type

Reply:
Addressed